### ISSUE TITLE
New problem plz help error: FetchError: request to http://api.nftport.xyz/v0/files failed, reason: socket hang up

### ISSUE THREAD

**Issue by TheGODMasterDu** [HashLips/hashlips_art_engine#863]
So I'm trying to run this code node utils/nftport/uploadFiles.js but I keep running into this.
![PencilNeck to Hashlips JS Utils Getting Close](https://user-images.githubusercontent.com/100662087/156790957-516bb081-ccf7-40e0-9a68-0a36283455b4.png)

**Comment by bolshoytoster** [comment]
@TheGODMasterDu codeSTACKr's repo is https://github.com/codeSTACKr/create-10k-nft-collection

**Comment by TheGODMasterDu** [comment]
> @TheGODMasterDu codeSTACKr's repo is https://github.com/codeSTACKr/create-10k-nft-collection

Yeah I've gone over that video a million times and it isn't working for me.

**Comment by bolshoytoster** [comment]
@TheGODMasterDu not the video, open an issue there because the error doesn't come from hashlips' code.
